src/style.css

```diff
@@ -1,3 +1,65 @@
+:root {
+  --blue: #00acdc;
+  --black: #000;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
```

Comment:
You should add a fallback font in case 'Roboto' doesn't load. This can be a generic font like 'sans-serif'. So it would be `font-family: Roboto, sans-serif;`. This is in line with the checklist item: 'Remember to use fallback fonts - alternative font-family in case the main one doesn't work'.